src/EnergyPlus/MixedAir.cc

```diff
@@ -1070,7 +1083,9 @@ namespace MixedAir {
                                           cNumericFields);
             UtilityRoutines::IsNameEmpty(AlphArray(1), CurrentModuleObject, ErrorsFound);
             OutsideAirSys(OASysNum).Name = AlphArray(1);
-            GlobalNames::IntraObjUniquenessCheck(AlphArray(2), CurrentModuleObject, cAlphaFields(2), ControllerListUniqueNames, ErrorsFound);
+            if (!UtilityRoutines::SameString(AlphArray(2), "")) {
```

Comment:
You can just use:

```
if (!AlphArray(2).empty()) {
```

to check if a string is blank.

Reply:
@Myoldmopar Done. Thanks.